ds_judgements_public_ui/templates/pages/how_to_use_this_service.html

```diff
@@ -0,0 +1,199 @@
+{% extends "base.html" %}
+{% load i18n static %}
+
+{% block title %}
+  {% translate "howtousethisservice.title" %} - {% translate "common.findcaselaw" %}
+{% endblock title %}
+
+{% block content %}
+  <header class="page-header">
+    {% include 'includes/breadcrumbs.html' with current=page_title title=page_title link=request.path %}
+    {% include 'includes/logo.html' %}
+  </header>
+
+  <div class="standard-text-template">
+    <h1>How to use the Find Case Law service</h1>
+    <p><small>Last updated on
+      <time datetime="27-05-2022">27th May 2022</time>
+    </small></p>
+
+    <div class="anchor-links">
+      <h3 class="anchor-links__header">Information on this page:</h3>
+      <ul class="anchor-links__list">
+        <li class="anchor-links__list-option"><a href="#section-about">What is case law</a></li>
+        <li class="anchor-links__list-option"><a href="#section-no-advice">We do not provide legal advice</a></li>
+        <li class="anchor-links__list-option"><a href="#section-find">Find a judgment or decision</a></li>
+        <li class="anchor-links__list-option"><a href="#section-understanding">Understanding a judgment or decision</a>
+        </li>
+        <li class="anchor-links__list-option"><a href="#section-contact">How to contact us</a></li>
+      </ul>
+    </div>
+
+    <h2 id="section-about">What is case law</h2>
+
+    <p>Case law is the law which is created through court judgments and tribunal decisions. This is sometimes known as
+      common law. Judges will follow the 'precedent' or the principles used in previous judgments and decisions when
+      deciding on a case. Case law can be used to clarify legislation if it is unclear how it applies in a particular
+      context.</p>
+    <p>Legislation is law enacted by Parliament or made by the government, and is sometimes known as statute law. Visit the <a href="https://www.legislation.gov.uk">legislation.gov.uk website</a> to find out more.</p>
+
+    <h2 id="section-no-advice">We do not provide legal advice</h2>
+
+    <p>This service is designed to help you find court judgments or tribunal decisions. The National Archives does not
+      provide legal advice, interpretation or support. You can find guidance and support at the following
+      organisations:</p>
+    <ul>
+      <li><a href="https://www.citizensadvice.org.uk">Citizens Advice Bureau</a></li>
+      <li><a href="https://www.lawcentres.org.uk">The Law Centres Network</a></li>
+      <li><a href="https://www.supportthroughcourt.org">Support Through Court</a></li>
+      <li><a href="https://www.weareadvocate.org.uk">Advocate</a></li>
+    </ul>
+
+
+    <h2 id="section-find">Find a judgment or decision</h2>
+    <p>You can search the full text of every judgment and decision. You can also search and browse judgments by neutral
+      citation, court, party, judge and within a date range. These options can be combined on the <a
+              href="/search">More search options</a> page.</p>
```

Comment:
This url doesn't exist -- did you mean `/structured_search` ?

Although I feel you're entirely right that `/search` is a far better url...

Reply:
Thanks @dragon-dxw - I've just changed this 👍🏽